src/mercury_engine_data_structures/version_validation.py

```diff
@@ -0,0 +1,62 @@
+from hashlib import md5
+
+from construct.core import BytesInteger, Int32ul, PrefixedArray, Struct
+
+from mercury_engine_data_structures.file_tree_editor import FileTreeEditor, GameVersion
+from mercury_engine_data_structures.formats.property_enum import FileNameEnum
+from mercury_engine_data_structures.formats.toc import Toc
+
+MANIFEST_CONSTRUCT = Struct(
+    "toc_md5" / BytesInteger(16),
+    "file_data" / PrefixedArray(Int32ul, Struct("assetid" / FileNameEnum, "md5" / BytesInteger(16))),
+)
+
+
+def get_md5(data: bytes) -> int:
+    return int(md5(data, usedforsecurity=False).hexdigest(), 16)
```

Comment:
This should instead just be `md5(data, usedforsecurity=False).digest()` with return value being bytes. And use bytes instead of ints to hold the hashes.